.travis.yml

```diff
@@ -26,6 +26,7 @@ matrix:
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
 
           # Test the latest stable release
+        - php: 7.4
```

Comment:
should be sorted in increasing order, and 7.4 the thing with coverage and whatnot.